.rubocop_todo.yml

```diff
@@ -12,3 +14,21 @@ Metrics/PerceivedComplexity:
 
 Style/FrozenStringLiteralComment:
   Enabled: false
+
+RSpec/InstanceVariable:
+  Enabled: false
```

Comment:
I'm totally in favor of this cop! I've always avoided ivars in my specs. It'll be nice to lint that.

It's ok to leave it `Enabled: false` for now. I'll turn it on later.